probzelus/inference/ds_naive_graph.ml

```diff
@@ -210,6 +210,9 @@ let get_distr_kind : type a b.
   | DSnaive_Initialized (_, CBernBern _) -> KBernoulli
   | DSnaive_Marginalized (Dist_bernoulli _, _) -> KBernoulli
   | DSnaive_Marginalized (Dist_beta _, _) -> KBeta
+  | DSnaive_Marginalized (Dist_binomial _, _) -> KOthers
+  | DSnaive_Marginalized (Dist_beta_binomial _, _) -> KOthers
+  | DSnaive_Marginalized (Dist_negative_binomial _, _) -> KOthers
   | DSnaive_Marginalized (( Dist_sampler _
```

Comment:
Don't you need to identify the distribution such that the `Ds_high_level` can find the conjugacy relation?